.github/workflows/ci.yml

```diff
@@ -2,10 +2,10 @@ name: CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [ trunk ]
     tags: [ "**" ]
   pull_request:
-    branches: [ master ]
+    branches: [ trunk ]
```

Comment:
Maybe allow PR  to any branch... to simplify the CI config.
```suggestion
```

Reply:
Yes, I do actually quite prefer that behavior, even aside from the simplification.